README.asciidoc

```diff
@@ -28,12 +28,23 @@ link:http://open.qa/docs/#_development_version_repository[openQA documentation].
 
 For building os-autoinst manually checkout the build instructions below.
 
-To execute an instance of os-autoinst one needs to create a file named
-`vars.json`. It stores the values of the different variables that will configure
-the behavior of the test execution.
+The main executable is `isotovideo` can read test parameters from the command
+line or read test parameters from a file named `vars.json`. This file stores
+the values of the different variables that will configure the behavior of the
+test execution.
 
-There are some variables used by os-autoinst itself and other that are
-used by the tests. A minimal `vars.json` file can be:
+
```

Comment:
```suggestion
```

Reply:
please check again if I did what you suggested. The diff showed something for me which I could not map to the current state of text I had locally. Maybe I forgot to push an update. Please check again.